frontend/src/__tests__/integration/pages/modelServing/ModelServingGlobal.stories.tsx

```diff
@@ -67,7 +67,7 @@ const getHandlers = ({
     (req, res, ctx) => res(ctx.json(mockK8sResourceList(inferenceServices))),
   ),
   rest.get('/api/k8s/api/v1/namespaces/test-project/secrets', (req, res, ctx) =>
-    res(ctx.json(mockK8sResourceList([mockSecretK8sResource({})]))),
+    res(ctx.json(mockK8sResourceList([mockDataConnection({ displayName: 'Other secret' }).data]))),
```

Comment:
Heh, if you change the secret name with the example we talked about it will break some test cases since they are expecting the default name, change one or another!